### PR TITLE
PSS Shuttles Fix + one extra tritium stack on Dominian corvette

### PR DESCRIPTION
**Pull request by lavillastrangiato**
title

### CODE REVIEW

**Comment by lavillastrangiato** [comment]
!review

File: maps/away/away_site/tajara/peoples_station/peoples_station.dmm [hunk 1]
```diff
@@ -328,11 +328,16 @@
...
 "bA" = (
+/obj/effect/map_effect/window_spawner/full/shuttle{
+	spawn_firedoor = 1;
+	spawn_grille = 1;
+	color = "#6C7364";
+	frame_color = "#6C7364"
+	},
 /obj/machinery/door/blast/regular/open{
-	id = "peoples_station_transport_blast";
+	id = "elyran_shuttle_shutters";
```
Comment (DreamySkrell) [inline maps/away/away_site/tajara/peoples_station/peoples_station.dmm:338]: ?

File: maps/away/away_site/tajara/peoples_station/peoples_station.dmm [hunk 2]
```diff
@@ -3835,9 +3842,16 @@
...
+	pixel_x = 24;
+	id_tag = "headmaster_shuttle";
+	child_names_txt = "Orbital Fleet Shuttle Airlock";
+	child_tags_txt = "headmaster_shuttle";
+	dir = 8
+	},
 /obj/machinery/button/remote/blast_door{
 	pixel_x = -5;
-	id = "peoples_station_transport_blast";
+	id = "elyran_shuttle_shutters";
```
Comment (DreamySkrell) [inline maps/away/away_site/tajara/peoples_station/peoples_station.dmm:3854]: ?

File: maps/away/away_site/tajara/peoples_station/peoples_station.dmm (hunk 1 above)
Comment (lavillastrangiato) [reply maps/away/away_site/tajara/peoples_station/peoples_station.dmm:338]: huh. is that still on there? i'll fix it soon

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**